net-analyzer/ospd-openvas/ospd-openvas-1.0.1.ebuild

```diff
@@ -20,8 +20,8 @@ DEPEND="
 	acct-user/gvm
 	dev-python/packaging[${PYTHON_USEDEP}]
 	dev-python/psutil[${PYTHON_USEDEP}]
-	dev-python/redis-py[${PYTHON_USEDEP}]
-	net-analyzer/ospd[${PYTHON_USEDEP}]
+	>=dev-python/redis-py-3.0.1[${PYTHON_USEDEP}]
+	>=net-analyzer/ospd-2.0.0[${PYTHON_USEDEP}]
```

Comment:
This would require a revbump if there were multiple choices present in the tree, but as this is totally useless change I don't mind it.